images/tidb-backup-manager/entrypoint.sh

```diff
@@ -17,6 +17,8 @@ set -e
 
 # The trap command is to make sure the sidecars are terminated when the jobs are finished
 cleanup() {
+    echo "Sleeping for 30 seconds before exit..."
```

Comment:
Can you please add a comment motivating the sleep.

Reply:
@nkg- added